qiskit/circuit/library/standard_gates/r.py

```diff
@@ -79,7 +79,7 @@ def _define(self):
         self.definition = qc
 
     def inverse(self, annotated: bool = False):
-        """Invert this gate as: :math:`r(θ, φ)^dagger = r(-θ, φ)`
+        """Invert this gate as: :math:`r(θ, φ)^{\dagger} = r(-θ, φ)`
```

Comment:
It's also referred to as uppercase `R` in the docs, could you update that as well? 🙂 
```suggestion
        """Invert this gate as: :math:`R(θ, φ)^{\dagger} = R(-θ, φ)`
```